### PR TITLE
Checkstyle errors fixed - GameState.java

### PR DESCRIPTION
**Pull request by peace-4-you**
Solved trailing spaces
Solved tab errors

### CODE REVIEW

File: bust-a-move-framework/src/main/java/nl/tudelft/bust_a_move20162017/bust_a_move_framework/gamestate/GameState.java [hunk 1]
```diff
@@ -10,33 +10,31 @@
...
-	 */
-	public static final int WIN_SCREEN = 5;
-	/**
-	 * Integer number of game state.
-	 */
-	public static final int DEFEAT_SCREEN = 6;
+    /**
+     * Integer number of game state.
+     */
+    public static final int START_SCREEN = 1;
```
Comment (CalvinNhieu) [inline bust-a-move-framework/src/main/java/nl/tudelft/bust_a_move20162017/bust_a_move_framework/gamestate/GameState.java]: *

File: bust-a-move-framework/src/main/java/nl/tudelft/bust_a_move20162017/bust_a_move_framework/gamestate/GameState.java [hunk 2]
```diff
@@ -10,33 +10,31 @@
...
+    /**
+     * Integer number of game state.
+     */
+    public static final int MAIN_MENU = 2;
+    /**
+     * Integer number of game state.
```
Comment (CalvinNhieu) [inline bust-a-move-framework/src/main/java/nl/tudelft/bust_a_move20162017/bust_a_move_framework/gamestate/GameState.java]: *

**Comment by peace-4-you** [comment]
> Possibly consider leaving *less generic member descriptions?

You mean removing the javadoc? If so, checkstyle will complain about it. If not, what do you suggest writing in the javadoc then ?

**Comment by CalvinNhieu** [comment]
@peace-4-you I mean, the javadoc is identical for every member. I was suggesting something like: "Integer number of main menu game state."

**Comment by peace-4-you** [comment]
> I mean, the javadoc is identical for every member. I was suggesting something like: "Integer number of main menu game state."

Done.

**Comment by mauricewillemsen** [comment]
Looks fine

**Pull request merged**